### ISSUE TITLE
When uploading a new job list, the session data are preserved, which can cause problems. 

### ISSUE THREAD

**Issue by nclaidiere** [open-cogsci/omm-server#114]
For instance, previously when I removed a participant I could restart the experiment for that participant, now I have to remove the entire experiment (therefore all the participants) and redo everything (assign participants, upload job list, etc). Would it be possible to wipe out the data?

**Comment by dschreij** [comment]
Yes, this basically means that when uploading a new job list, all session data needs to be reset?
Also be aware that by uploading a new job list, you will erase all the already collected data for the experiment (the previous job list)

**Comment by nclaidiere** [comment]
_when uploading a new job list, all session data needs to be reset?_

Yes since the job data are removed anyway I think its best to remove everything, resetting the experiment to 0. This is important when you are developing the experiment and testing it.

**Comment by dschreij** [comment]
Even though you marked this as wontfix, I think it's a minor change which I can do at some point. I'll prioritize the other issues though.
